config/database.pg.yml

```diff
@@ -15,7 +15,7 @@ base: &base
   adapter: postgresql
   encoding: utf8
   username: root
-  pool: 1
+  pool: 5

```

Comment:
> I'm ok with changing this as long as rails times out unused connections

It doesn't -- though we could fix that.

> Each pg postmaster client process on appliances consume around 100 MB of memory or more

That sounds remarkably high, but.. okay.

---

It sounds like we'll need to do something different.

@imtayadeway we probably need to arrange to release the DB connection in the master thread, when the application's finished booting & before it starts serving requests. As a first guess, try just doing that in an initializer that'll run last: `config/initializers/zzz_disconnect_database.rb`?

Otherwise, we may need to use puma's "I'm about to spawn a new thread" hook.